tests/python/unittest/test_operator.py

```diff
@@ -6913,6 +6913,28 @@ def test_invalid_depth_dim():
     test_invalid_block_size()
     test_invalid_depth_dim()
 
+@with_seed()
+def test_correlation_kernel_size():
+    import numpy as np
```

Comment:
I think these are already imported in this module

Reply:
True. My bad. Removing it.